CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added assert to NetCDF4_get_var.H to print variable name if data retrieval fails
 
+- Added MAPL_GetGlobalHorzIJIndex subroutine
```

Comment:
Git diff can provide this sort of information.   What is needed here is a description of the capability that the procedure provides.   Still useful to name the actual procedure of course, and yes, the name is somewhat indicative, but ...

Reply:
changed